js/manager.js

```diff
@@ -182,6 +187,21 @@ const DFPManager = Object.assign(new EventEmitter().setMaxListeners(0), {
     }
   },
 
+  customLoad(slots, options = {}) {
```

Comment:
Instead of having a new+different `customLoad` method I wonder if we should consider the following interface:

```
DFPManager.configureLimitedAds(true);
DFPManager.load();
```

For example, this is how we configure the the singleRequest flag:
https://github.com/jaanauati/react-dfp/pull/209/files#diff-718ec08e33477933ddc079405f59eb8188ea4a29dbb8547d7e57cbe00d579b6bR170

Does it make sense?

Reply:
Yes indeed, you're right !
There is no reason to mess with the `load(...)` function
=> https://github.com/jaanauati/react-dfp/pull/209/commits/2e2d88b868e2fc3e8c282cbb6e2ca508d7faeb19

Reply:
Hello @jaanauati 

Are the changes OK ?
Do you want me to check anything else ?